chart/operator/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 apiVersion: v2
-name: skywalking-swck-operator
+name: skywalking-helm-swck-operator
```

Comment:
We will publish these to hub.docker.io, and to make it clear this is a Helm Chart, not a Docker image, I change the names to be prefixed with `skywalking-helm-*`, and will ask infra team to create the repositories with the same name prefix

Reply:
So, this is a helm for swck?

Reply:
> So, this is a helm for swck?

Yes, as the title indicates. swck contains two Helm Charts actually, namely, operator and adapter

Reply:
Can't dockerhub directly know that this is a chart type?

Reply:
> Can't dockerhub directly know that this is a chart type?

Of course it knows, I meant I want to make it clear for our users.

Reply:
@innerpeacez I think here is about avoiding confusion of another existing repo, https://hub.docker.com/r/apache/skywalking-swck

Reply:
got it.